platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/AbstractDomDayPopupButton.java

```diff
@@ -37,7 +37,11 @@ public AbstractDomDayPopupButton() {
 
 	private void updateLabel() {
 
-		setLabel(IDisplayString.create(showLabel? chooser.getDay().toISOString() : ""));
+		if (showLabel) {
```

Comment:
I think we can revert the changes to this file. Should not make a difference since the change in DomButton was introduced.

Reply:
this change makes this class more readable imho, without having to know that `setLabel("")` removes the label